modules/op-setting-up-openshift-pipelines-to-view-project-vulnerabilities.adoc

```diff
@@ -146,3 +137,8 @@ spec:
 * Navigate to the `PipelineRun` details page and review the *Vulnerabilities* row for a visual representation of identified vulnerabilities.
 
 * Alternatively, you can navigate to the `PipelineRun` list view page, and review the *Vulnerabilities* column.
+
+[role="_additional-resources"]
+.Additional resources
+
+link:https://developers.redhat.com/products/trusted-application-pipeline/overview[Red Hat Trusted Application Pipeline (RHTAP)] combines OpenShift pipelines with other Red Hat products to deliver a secure, automated, and customizable end-to-end solution for building applications. Elevate your development process with RHTAP's integrated DevSecOps CI/CD framework, ensuring robust security and streamlined efficiency from inception to deployment. To get started, refer link:https://docs.redhat.com/en/documentation/red_hat_trusted_application_pipeline/1.0/html/getting_started_with_red_hat_trusted_application_pipeline/index[Getting Started with Red Hat Trusted Application Pipeline]
```

Comment:
So as this stands, this additional resource section has too much text outside of the link text, which goes against the [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections). It should also present the links in an unordered list.

If the text outside the link is critical to include in your opinion, maybe there are other ways to introduce these links elsewhere in this content. In any case, your additional resources should look more like this:
```suggestion
* link:https://developers.redhat.com/products/trusted-application-pipeline/overview[Red Hat Trusted Application Pipeline (RHTAP)] 
* link:https://docs.redhat.com/en/documentation/red_hat_trusted_application_pipeline/1.0/html/getting_started_with_red_hat_trusted_application_pipeline/index[Getting Started with Red Hat Trusted Application Pipeline]
```

Reply:
Went with a bit of long sentences as here - https://docs.openshift.com/container-platform/4.16/edge_computing/policygenerator_for_ztp/ztp-advanced-policygenerator-config.html#ztp-using-pgt-to-configure-power-saving-mode_ztp-advanced-policygenerator-config